### PR TITLE
update vector to 0.11.1 version in build clusters

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 
Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: clusters/build-clusters/01_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: vector-audit-log
-        image: docker.io/timberio/vector:0.9.0-alpine
+        image: docker.io/timberio/vector:0.11.1-alpine
```
Comment (alvaroaleman) [inline clusters/build-clusters/01_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml:25]: What is the reason we have multiple manifests for this?

File: clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml [hunk 2]
```diff
@@ -6,3 +6,39 @@ metadata:
     openshift.io/description: This SA is used for uploading audit-log by vector
   name: logging-app
   namespace: api-audit-log
+---
+apiVersion: security.openshift.io/v1
```
Comment (alvaroaleman) [inline clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml:10]: Why do we need this now, the manifest is unchanged?

Comment (droslean) [reply clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml:10]: Nope... that was always needed. I don't have any clue how it worked before. I guess we applied the scc manually?

File: clusters/build-clusters/01_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml (hunk 1 above)
Comment (droslean) [reply clusters/build-clusters/01_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml:25]: No idea. I think @hongkailiu added those manifests. Shall we put them all in one file?

File: clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml (hunk 2 above)
Comment (alvaroaleman) [reply clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml:10]: So it already existed and just wasn't in git?

Comment (droslean) [reply clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_logging-app_rbac.yaml:10]: Yes. Here I add a specific scc for the `logging-app` service account.

**Pull request merged**